src/Uno.UI/UI/Xaml/Style/Generic/SystemResources.xaml

```diff
@@ -791,7 +791,7 @@
 			<StaticResource x:Key="TimePickerButtonForegroundFocused" ResourceKey="SystemControlHighlightAltBaseHighBrush" />
 
 			<!-- Resources for Windows.UI.Xaml.Controls.TimePickerFlyoutPresenter -->
-			<StaticResource x:Key="TimePickerFlyoutPresenterBackground" ResourceKey="SystemControlTransientBackgroundBrush" />
+			<StaticResource x:Key="TimePickerFlyoutPresenterBackground" ResourceKey="SystemChromeBlackHighColor" />
```

Comment:
you shouldn't be mapping `TimePickerFlyoutPresenterBackground` to different system resources for different themes. Do you really need this?